### PR TITLE
Rename 'flip'  to 'turn over'

### PR DESCRIPTION
**Pull request by tooomm**
## Related Ticket(s)
- Related to #3094

## Short roundup of the initial problem
`Flip` was used as a potentially confusing term for "turning a card face up/down"

## What will change with this Pull Request?
- Use `Turn over` instead

### CODE REVIEW

File: cockatrice/src/player.cpp [hunk 1]
```diff
@@ -733,7 +733,7 @@ void Player::setShortcutsActive()
     aPlay->setShortcuts(settingsCache->shortcuts().getShortcut("Player/aPlay"));
     aTap->setShortcuts(settingsCache->shortcuts().getShortcut("Player/aTap"));
     aDoesntUntap->setShortcuts(settingsCache->shortcuts().getShortcut("Player/aDoesntUntap"));
-    aFlip->setShortcuts(settingsCache->shortcuts().getShortcut("Player/aFlip"));
+    aTurnOver->setShortcuts(settingsCache->shortcuts().getShortcut("Player/aTurnOver"));
```
Comment (tooomm) [inline cockatrice/src/player.cpp]: This string looks weird to me. What is exactly used for?

Comment (ctrlaltca) [reply cockatrice/src/player.cpp]: Settings are saved to an ini file; that's the "key" used to identify that specific setting in the file, eg:.
```
[Custom]
...
Player\aDrawCards=Ctrl+E
```

File: cockatrice/src/player.cpp [hunk 2]
```diff
@@ -682,9 +682,11 @@ void Player::retranslateUi()
     aPlay->setText(tr("&Play"));
     aHide->setText(tr("&Hide"));
     aPlayFacedown->setText(tr("Play &Face Down"));
+    //: Turn sideways or back again
     aTap->setText(tr("&Tap / Untap"));
     aDoesntUntap->setText(tr("Toggle &normal untapping"));
-    aFlip->setText(tr("&Flip"));
+    //: Turn face up/face down
+    aTurnOver->setText(tr("&Turn Over"));
```
Comment (ZeldaZach) [inline cockatrice/src/player.cpp]: This won't work; We have 2 &T's now. I honestly dislike the windows automated shortcuts as they cause translations to be a PITA

Comment (tooomm) [reply cockatrice/src/player.cpp]: Good catch! Will change that.

Is it windows only? I think it's a general way to access menus on all platforms?

But wait... does this work for right click menus at all?
For the main menus at top sure, no problem. But here I didn't get it to work in a local test game.

Any ideas? Could we remove it from all right click menus?

**Comment by tooomm** [comment]
>Is this going to break server protocol? 

I tested it live on the server (offline it works), you are right! @ZeldaZach. 😞 

The renamed `.proto` files are the problem I assume?

I can change that back for now, but at one point we need to gather such updates and the servers need a new feature requirement. Otherwise the code will fall apart more and more.
Or do we opt for the string change only? But still, that makes it difficult for new contributors...

File: common/pb/CMakeLists.txt [hunk 3]
```diff
@@ -23,7 +23,7 @@ SET(PROTO_FILES
     command_delete_arrow.proto
     command_draw_cards.proto
     command_dump_zone.proto
-    command_flip_card.proto
+    command_turn_card_over.proto
```
Comment (Daenyth) [inline common/pb/CMakeLists.txt]: Don't change the proto

Comment (tooomm) [reply common/pb/CMakeLists.txt]: So do we keep track of such left out changes and do them at one point combined?

In addition to requesting the server admins to require that "feature" when connecting to their servers?
Or bumping Cockatrice to v3 as incompatible change?

File: common/pb/game_commands.proto [hunk 4]
```diff
@@ -9,7 +9,7 @@ message GameCommand {
         ROLL_DIE = 1005;
         DRAW_CARDS = 1006;
         UNDO_DRAW = 1007;
-        FLIP_CARD = 1008;
+        TURN_CARD_OVER = 1008;
```
Comment (tooomm) [inline common/pb/game_commands.proto]: These proto changes should be save because they are digit coded?

File: common/pb/CMakeLists.txt (hunk 3 above)
Comment (Daenyth) [reply common/pb/CMakeLists.txt]: Add a comment here, but otherwise leave all the proto files alone.

This should only be a client-facing string change.

**Comment by ZeldaZach** [comment]
Honestly, changing all parts of the code (internal) is not really necessary. Provided we just change the visible string (external) I feel that is sufficient

**Comment by ZeldaZach** [comment]
Do we have any plans with this PR? I don't fully agree with it and it's going to cause issues between what protocol is and what we think the protocol actually is with the namings

**Comment by tooomm** [comment]
Ok. So https://github.com/Cockatrice/Cockatrice/pull/3100#discussion_r167368554 and https://github.com/Cockatrice/Cockatrice/pull/3100#issuecomment-364630117 it is? UI only change in the client with a code comment for the protocol that we are renaming internally with a v3 protocol rewrite?

I can submit the changes tomorrow.

**Comment by tooomm** [comment]
see #3149

**Pull request closed without merging**